xpk.py

```diff
@@ -2624,6 +2624,107 @@ def run_gke_cluster_create_command(
   return 0
 
 
+def run_gke_pathways_cluster_update_command(args) -> int:
+  """Run the Pathways cluster update command for existing clusters.
+
+  Args:
+    args: user provided arguments for running the command.
+
+  Returns:
+    0 if successful and 1 otherwise.
+  """
+  command = (
+      'gcloud container clusters update'
+      f' {args.cluster} --project={args.project}'
+      f' --region={zone_to_region(args.zone)}'
+      ' --cluster-dns=clouddns'
+      ' --cluster-dns-scope=vpc'
+      f' --cluster-dns-domain={args.cluster}-domain'
+      ' --quiet'
+  )
+  xpk_print('Updating Pathways cluster to use Cloud DNS, may take a while!')
+  return_code = run_command_with_updates(
+      command, 'Pathways Cluster Update to enable Cloud DNS', args
+  )
+  if return_code != 0:
+    xpk_print(f'Pathways Cluster Update request returned ERROR {return_code}')
+    return 1
+  return 0
```

Comment:
Remove Pathways from these logs since these actions are not pathways specific but help enable a feature on the pathways stack